src/libraries/System.Net.Security/tests/FunctionalTests/TestHelper.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Sockets;
+using System.Net.Test.Common;
+
+namespace System.Net.Security.Tests
+{
+    public static class TestHelper
+    {
+        public static (Stream ClientStream, Stream ServerStream) GetConnectedStreams()
+        {
+            if (Capability.SecurityForceSocketStreams())
+            {
+                return GetConnectedTcpStreams();
+            }
+
+            return GetConnectedVirtualStreams();
+        }
+
+        internal static (NetworkStream ClientStream, NetworkStream ServerStream) GetConnectedTcpStreams()
+        {
+            using (Socket listener = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp))
```

Comment:
nit:

You might want to use the new C# 'using' syntax feature as it makes the code look cleaner.

```c#
using Socket listener = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
listener.Bind(new IPEndPoint(IPAddress.Loopback, 0));
listener.Listen(1);

clientSocket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
clientSocket.Connect(listener.LocalEndPoint);
Socket serverSocket = listener.Accept();

return (new NetworkStream(clientSocket, ownsSocket: true), new NetworkStream(serverSocket, ownsSocket: true));
```

Reply:
> You might want to use the new C# 'using' syntax feature as it makes the code look cleaner.

There's certainly room for debate there :smile:.  I find it way _less_ clean (since there's still a scope, but it's implicit instead of explicit).